homeassistant/components/circuit/__init__.py

```diff
@@ -0,0 +1,53 @@
+"""The Unify Circuit component."""
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_NAME, CONF_URL
+from homeassistant.helpers import config_validation as cv, discovery
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "circuit"
+CONF_WEBHOOK = "webhook"
+
+_WEBHOOK_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_NAME): cv.string,
+        vol.Required(CONF_URL): vol.All(cv.ensure_list, [cv.string]),
+    }
+)
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.All(
+            cv.ensure_list,
+            [
```

Comment:
Do we need a list here and a list for the webhook schema?

Reply:
The list for the CONF_URL is needed so we can submit the same notify message to multiple conversations within Circuit. The list for the CONFIG_SCHEMA I'll change that. New commit in PR will be submitted.

See configuration.yaml entries to clarify
```
# Example configuration.yaml entry
circuit:
  - webhook:
      - name: webhook_1
        url: https://eu.yourcircuit.com/rest/v2/webhooks/incoming/<ID-1>
      - name: webhook_2
        url: https://eu.yourcircuit.com/rest/v2/webhooks/incoming/<ID-2>
      - name: webhook_multiple
        url: 
          - https://eu.yourcircuit.com/rest/v2/webhooks/incoming/<ID-1>
          - https://eu.yourcircuit.com/rest/v2/webhooks/incoming/<ID-2>
```